### ISSUE TITLE
[Bug]: 2.1.9 and Node DB

### ISSUE THREAD

**Issue by tb0hdan** [meshtastic/firmware#2446]
### Category

Other

### Hardware

T-Beam

### Firmware Version

2.1.9

### Description

After reaching 75 nodes total in Node DB device stops responding even via serial.

```
$ meshtastic --reboot
Traceback (most recent call last):
  File "/usr/local/bin/meshtastic", line 8, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.9/dist-packages/meshtastic/__main__.py", line 1335, in main
    common()
  File "/usr/local/lib/python3.9/dist-packages/meshtastic/__main__.py", line 945, in common
    client = meshtastic.serial_interface.SerialInterface(
  File "/usr/local/lib/python3.9/dist-packages/meshtastic/serial_interface.py", line 61, in __init__
    StreamInterface.__init__(
  File "/usr/local/lib/python3.9/dist-packages/meshtastic/stream_interface.py", line 50, in __init__
    self.connect()
  File "/usr/local/lib/python3.9/dist-packages/meshtastic/stream_interface.py", line 74, in connect
    self._waitConnected()
  File "/usr/local/lib/python3.9/dist-packages/meshtastic/mesh_interface.py", line 524, in _waitConnected
    raise Exception("Timed out waiting for connection completion")
Exception: Timed out waiting for connection completion
```

Still, after a bunch of tries, I was able to run 

```
meshtastic --factory-reset
```

And the device became alive again.

Still, there are other issues, like permanently loosing WiFi connectivity (doesn't even ping) with logfile *flooded* with:

```
[683525][I][WiFiClient.cpp:549] connected(): Unexpected: RES: 0, ERR: 12
[683533][E][WiFiUdp.cpp:185] endPacket(): could not send data: 12
[683537][I][WiFiClient.cpp:549] connected(): Unexpected: RES: 0, ERR: 12
[683545][E][WiFiUdp.cpp:185] endPacket(): could not send data: 12
[683550][E][WiFiUdp.cpp:185] endPacket(): could not send data: 12
[683557][E][WiFiUdp.cpp:185] endPacket(): could not send data: 12
[683562][E][WiFiUdp.cpp:185] endPacket(): could not send data: 12
[683568][E][WiFiUdp.cpp:185] endPacket(): could not send data: 12
[683573][E][WiFiUdp.cpp:185] endPacket(): could not send data: 12
[683579][E][WiFiUdp.cpp:185] endPacket(): could not send data: 12
[683586][E][WiFiUdp.cpp:185] endPacket(): could not send data: 12
[683591][E][WiFiUdp.cpp:185] endPacket(): could not send data: 12
[683597][E][WiFiUdp.cpp:185] endPacket(): could not send data: 12
[683603][E][WiFiUdp.cpp:185] endPacket(): could not send data: 12
[683608][E][WiFiUdp.cpp:185] endPacket(): could not send data: 12
[683615][I][WiFiClient.cpp:549] connected(): Unexpected: RES: 0, ERR: 12
```

Rolled back to 2.1.7 for the time being.

**Comment by garthvh** [comment]
Where you able to get to 80 nodes in 2.1.7?

**Comment by tb0hdan** [comment]
No, it just reboots and keeps working after that

**Comment by thebentern** [comment]
This is resolved in 2.2.X firmware variants
